Vates/VatesAPI/test/SQWLoadingPresenterTest.h

```diff
@@ -51,38 +53,51 @@ void setUp()
 
 void testConstructWithEmptyFileThrows()
 {
-  TSM_ASSERT_THROWS("Should throw if an empty file string is given.", SQWLoadingPresenter(new MockMDLoadingView, ""), std::invalid_argument);
+  std::unique_ptr<MDLoadingView> view =
+      Mantid::Kernel::make_unique<MockMDLoadingView>();
```

Comment:
Note that this slightly changes the test behavior: The allocation of `MDLoadingView` is not no longer inside the `ASSERT`. As far as I understand that is probably fine (i.e., not what we want to test), but maybe you can think about it for a minute to verify.
Same goes for most other tests in this file.